README.md

```diff
@@ -21,15 +21,11 @@ Most fields are `omitempty` because they disappear in certain situations. This c
 ## Usage
 
```

Comment:
Kind of a nitpicky thing, but I feel like it makes sense to preface the README with some content about 

* What the goal of the project is
* Some anticipated use cases

That'll pair well with the usage and give people a starting point

Reply:
added these, thanks